internal/config/testdata/invalid_log_rotate_config_1.yaml

```diff
@@ -0,0 +1,7 @@
+write:
+  create-empty-file: true
+logging:
+  severity: error
+  log-rotate:
+    max-file-size-mb: -1
+    file-count: -1
```

Comment:
Just curious, what happens if someone assign float value on these options.

Reply:
It takes the floor integer value in this case.